waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/compatibility/HiveCompatibleThriftHiveMetastoreIfaceFactory.java

```diff
@@ -35,7 +35,7 @@ public class HiveCompatibleThriftHiveMetastoreIfaceFactory {
   private static class ThriftMetaStoreClientInvocationHandler implements InvocationHandler {
 
     private final ThriftHiveMetastore.Client delegate;
-    private final HiveThriftMetaStoreIfaceCompatibility compatibility;
+    private final HiveThriftMetaStoreIfaceCompatibility1x compatibility;
```

Comment:
Why here is using 1x instead of using the combination of both interfaces? 1x and 2x?

Reply:
that's wrong good find I'll have a look at that and try to write a test to capture it.